index/secondary_index_manager.cc

```diff
@@ -388,4 +395,17 @@ std::optional<std::unique_ptr<custom_index>> secondary_index_manager::get_custom
     return (*custom_class_factory)();
 }
 
+index_metrics::index_metrics(const sstring& ks_name, const sstring& index_name)
+    : ks_name(ks_name)
+    , query_latency() {
+    metrics.add_group("index_metrics",
+            {seastar::metrics::make_histogram(
+                    fmt::format("query_latencies"),
+                    [this]() {
+                        return to_metrics_histogram(query_latency);
+                    },
+                    seastar::metrics::description("Histogram of the latencies of index queries"))(idx_name(fmt::format("{}.{}", ks_name, index_name)))
```

Comment:
split it into two labels, one for ks, one for index_name (it's ok to call it something shorter like index or idx.